pkg/kubelet/volumemanager/volume_manager.go

```diff
@@ -228,7 +228,9 @@ func NewVolumeManager(
 		mounter,
 		hostutil,
 		volumePluginMgr,
-		kubeletPodsDir)
+		kubeletPodsDir,
+		podStateProvider,
```

Comment:
we'd already plumbed podStateProvider into NewVolumeManager, this just passes it on to the reconciler to use

Reply:
ok